workflow-templates/storybook.yml

```
@@ -0,0 +1,29 @@
+name: Deploy storybook on gh-pages
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy-website:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Use Node.js 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: npm ci
+      - name: Build storybook
+        run: npm run build-storybook
+      - name: Deploy to gh-pages
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
```

With a custom token, we can publish new sites immediately. Otherwise we need to push once manually to trigger the github pages builder.